carrier_shipping_label_template/views/report_shipping_label.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+        <record id="report_qweb_paperformat_ups" model="report.paperformat">
+            <field name="name">100 x 150 mm thermal label (e.g. Zebra GK420D)</field>
+            <field name="default" eval="True" />
+            <field name="format">custom</field>
+            <field name="page_height">150</field>
+            <field name="page_width">100</field>
+            <field name="orientation">Portrait</field>
+            <field name="margin_top">3</field>
+            <field name="margin_bottom">0</field>
+            <field name="margin_left">2</field>
+            <field name="margin_right">0</field>
+            <field name="header_line" eval="False" />
+            <field name="header_spacing">35</field>
+            <field name="dpi">90</field>
+        </record>
+
+        <record id="action_report_shipping_label" model="ir.actions.report">
+            <field name="name">Shipping Label</field>
+            <field name="model">stock.picking</field>
+            <field name="report_type">qweb-pdf</field>
+            <field
+      name="report_name"
+    >carrier_shipping_label_template.report_shipping_label</field>
+            <field
+      name="report_file"
+    >carrier_shipping_label_template.report_shipping_label</field>
+            <field name="binding_type">report</field>
+            <field
+      name="paperformat_id"
+      ref="carrier_shipping_label_template.report_qweb_paperformat_ups"
+    />
```

Comment:
The formatting looks again very strange - is this something your formatter does do on its own?

Reply:
@manengel I've modified for the xml formation.